include/logger_impl.h

```diff
@@ -46,7 +28,7 @@ namespace diskann {
     char*              _buf;
     int                _bufIndex;
     std::mutex         _mutex;
-    ANNIndex::LogLevel _logLevel;
+    LogLevel _logLevel;
```

Comment:
please run clang-format on files being checked in, or use VS built-in formatter that recognizes clang-format directives

Reply:
yes, i ran it on all the files. The index.cpp have some more lines of changes in formatting.